tests/admin-unlock/main.yml

```diff
@@ -45,7 +45,7 @@
 
 - name: Ostree Admin Unlock - Setup
   hosts: all
-  become: yes
+  become: "yes"
```

Comment:
Weird, why?

Reply:
yamllint complains -- warning  truthy value is not quoted  (truthy)

This is the rule: http://yamllint.readthedocs.io/en/latest/rules.html#module-yamllint.rules.truthy

Reply:
Ah and I see: http://yaml.org/type/bool.html

But something I don't quite understand here; the Ansible docs say the `become` flag is a boolean, so we shouldn't be converting it to a string AIUI.  

Or oh ugh, do we really need `!!bool true`?

Reply:
I think we can avoid the quoting by just using `become: true`

Reply:
```
$ cat test.yml 
---
become: true
$ yamllint test.yml && echo $?
0
```

Reply:
This is in the same area as #299

Reply:
I'll move it over to true/false

Reply:
Ah true, looks like the same issue.